src/pages/SonarFeed/SonarFeedActivityPage.js

```diff
@@ -38,8 +38,8 @@ const SonarFeedActivityPage = ({ activities, isLoading, isError, isEmpty }) => {
   }
   return activities ? (
     <div className={styles.wrapper}>
-      {activities.map(activity => (
-        <Fragment key={activity.triggeredAt}>
+      {activities.map((activity, index) => (
+        <Fragment key={index}>
```

Comment:
@Partysun @GarageInc Could `activities` be sorted? If they are, then using `index` as key is a bad practice

Reply:
@DmitriVanGuard Should be sorted by date. First item is the last activity.
But it can be fixed in a separate PR. Cause full view of activities must be made with our design, not as we do now.